transport/http/inbound.go

```diff
@@ -154,3 +155,16 @@ func (i *Inbound) Addr() net.Addr {
 
 	return listener.Addr()
 }
+
+// Introspect returns the state of the inbound for introspection purposes.
+func (i *Inbound) Introspect() introspection.InboundStatus {
```

Comment:
No, it just prevents external implementors from making their transports introspectable (until some future point-release).